<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GestorComentarios</class>
 <widget class="QDockWidget" name="GestorComentarios">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>418</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QTextEdit" name="txt_comentario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>371</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comentarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_comentar">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>240</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Comentar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_verComentarios">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>280</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver comentarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_verArbolComentarios">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>240</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Ver Ã¡rbol comentarios</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_regresar">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>280</y>
      <width>131</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Regresar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
